File: node/trello/lists/read.js
var Read = {
  init: function(configFileName)
  {
    require('../base/base').init(Read, configFileName);
    Read.baseUrl = Read.config.getApi('lists', 'item');
  },

  info: function(id)
  {
    Read.trello.getData(
      Read.baseUrl + '/' + id,
      '&fields=name,closed',
      function(info) {
        if (info) {
          console.log('名称：%s（%s）：%s', info.name, info.id);
          console.log('关闭：%s', info.closed ? '是' : '否');
        }
      }
    );
  },

  cards: function(id)
  {
    Read.trello.getData(
      Read.baseUrl + '/' + id + '/cards',
      '&fields=name,closed',
      function(cards) {
        if (cards) {
          for (var index in cards) {
            console.log(
              '%s（%s）：%s',
              cards[index].name, cards[index].id,
              cards[index].closed ? '已关闭' : '未关闭'
            );
          }
        }
      }
    );
  }
};

module.exports = Read;

File: python/trello/lists/read.py
#-*- coding:utf-8 -*-
from trello.base.base import Base

class Read(Base):
  def __init__(self, configFileName):
    super(Read, self).__init__(configFileName)
    self.baseUrl = self.config.getApi('lists', 'item')

  def info(self, id):
    info = self.trello.getData(
      '%s/%s' % (self.baseUrl, id),
      '&fields=name,closed'
    )
    if info:
      print(
        '名称：%s（%s）' % (info['name'], info['id'])
      )
      print('关闭：%s' % ('是' if info['closed'] else '否'))

  def cards(self, id):
    cards = self.trello.getData(
      '%s/%s/cards' % (self.baseUrl, id),
      '&fields=name,closed'
    )
    if cards:
      for card in cards:
        print(
          '%s（%s）：%s' %
          (
            card['name'], card['id'],
            '已关闭' if card['closed'] else '未关闭'
          )
        )
